src/dfs/dfs_solve_new.cpp

```diff
@@ -235,14 +235,15 @@ template void tridiag_solver<Real>(view_2d<Real> AA, view_1d<Complex> name);
         Int nrows = rhse.extent(0);
         Int mid = ncols/2;
 
-        view_2d<Real> Llo_k("Lok", nrows, nrows);
-        view_2d<Real> Lle_k("Lek", nrows, nrows);
-        view_1d<Complex> fo("fo", nrows);
-        view_1d<Complex> fe("fe", nrows);
+        
 
        
         Kokkos::parallel_for(mid, [=](Int k){
                 Real dd =- pow(double(mid - k),2);
+                view_2d<Real> Llo_k("Lok", nrows, nrows);
```

Comment:
We can't declare new `Kokkos::View`s inside a kernel; Views must be allocated outside of a kernel, and then copied into the kernel.   If you truly need to allocate memory in the kernel and you don't know its size until runtime (i.e., not at compile time), then we need to allocate shared scratch memory for the thread team to use inside the kernel (but this is different than a View).